main.py

```
@@ -1,609 +1,802 @@
+    watch_session.update({
+        "active": True,
+        "changed_files": set(),
+        "thread": None,
+        "stop_event": stop_event,
+        "path": resolved,
+        "end_time": end_time,
+        "results": None
+    })
+    # At session start, store current content of all files in the watched path
+    previous_file_contents = {}
+    for root, dirs, files in os.walk(resolved):
```

Ruff (0.15.10)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)